wagtail/test/testapp/migrations/0026_featurecompletetoy.py

```diff
@@ -14,12 +15,12 @@ class Migration(migrations.Migration):
             name="FeatureCompleteToy",
             fields=[
                 (
-                    "id",
-                    models.AutoField(
-                        auto_created=True,
+                    "strid",
+                    models.CharField(
                         primary_key=True,
+                        default=wagtail.test.testapp.models.random_quotable_pk,
+                        max_length=255,
                         serialize=False,
```

Comment:
Not sure if this should be in a new migration, I don't think it's worth it because the testapp is pretty much ephemeral and not intended to be long-lived. Properly migrating this would need more than one migration (to add the new field, populate, remove the old one).

We could start with a new model, but I intended this model to be the main model to test all `ModelViewSet` features (hence "feature complete").